pandas/tseries/index.py

```diff
@@ -1458,7 +1460,9 @@ def _set_freq(self, value):
     dayofweek = _field_accessor('dayofweek', 'dow',
                                  "The day of the week with Monday=0, Sunday=6")
     weekday = dayofweek
+    day_of_week = dayofweek
     dayofyear = _field_accessor('dayofyear', 'doy', "The ordinal day of the year")
+    day_of_year = dayofyear
     quarter = _field_accessor('quarter', 'q', "The quarter of the date")
```

Comment:
let's switch the impl to the new names (and make the original ones aliases)
that way they are easy to remove